### ISSUE TITLE
Is there a way to change the arrow background and text color?

### ISSUE THREAD

**Issue by lkimuk** [oh-my-fish/theme-bobthefish#362]
![image](https://github.com/oh-my-fish/theme-bobthefish/assets/70332158/0e8b852e-5015-4351-bda4-cf7d557642a0)

**Comment by bobthecow** [comment]
Did you check out the [color schemes section of the readme](https://github.com/oh-my-fish/theme-bobthefish?tab=readme-ov-file#color-schemes), or the [custom color schemes page in the wiki](https://github.com/oh-my-fish/theme-bobthefish/wiki/Custom-color-schemes)?

**Comment by lkimuk** [comment]
Yeah, I'm completely new to fishshell. I've check it out, but there is not a `bobthefish_colors.fish` file in `~/.config/fish/functions/`. Should I create one?

**Comment by bobthecow** [comment]
Yes! I would urge you to read the [custom color schemes wiki page](https://github.com/oh-my-fish/theme-bobthefish/wiki/Custom-color-schemes), which answers that exact question :)

**Comment by lkimuk** [comment]
It's already kicking in! Tysm:)
